app/src/main/java/com/android/sample/ui/createRecipe/AddInstructionStepScreen.kt

```diff
@@ -0,0 +1,195 @@
+      navigationActions = navigationActions,
+      selectedItem = Route.CREATE_RECIPE,
+      showBackArrow = true,
+      content = { paddingValues ->
+        AddInstructionStepContent(
+            paddingValues = paddingValues,
+            createRecipeViewModel = createRecipeViewModel,
+            navigationActions = navigationActions)
+      })
+}
+
+/**
+ * Composable function that displays the content for adding an instruction step.
+ *
+ * @param paddingValues Padding values to be applied to the content.
+ * @param createRecipeViewModel ViewModel for managing the recipe creation process.
+ * @param navigationActions Actions for navigating between screens.
+ * @param modifier Modifier to be applied to the content.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AddInstructionStepContent(
+    paddingValues: PaddingValues,
+    createRecipeViewModel: CreateRecipeViewModel,
+    navigationActions: NavigationActions,
+    modifier: Modifier = Modifier
+) {
+  var stepDescription by remember { mutableStateOf("") }
+  var stepTime by remember { mutableStateOf("") }
+  var stepCategory by remember { mutableStateOf("") }
+  var selectedIcon by remember { mutableStateOf<IconType?>(null) }
+  var showError by remember { mutableStateOf(false) }
+
+  Column(
+      modifier =
+          modifier.fillMaxSize().padding(paddingValues).padding(horizontal = HORIZONTAL_PADDING.dp),
+      horizontalAlignment = Alignment.CenterHorizontally,
+      verticalArrangement = Arrangement.Top) {
+        // Step Label
+        Text(
+            text = stringResource(R.string.step_label),
+            style = Typography.titleLarge,
+            color = MaterialTheme.colorScheme.onPrimary,
+            modifier = Modifier.padding(vertical = 8.dp).testTag("StepLabel"))
+
+        // Container for input fields
+        Box(
+            modifier =
+                Modifier.fillMaxWidth()
+                    .background(color = lightCream, shape = RoundedCornerShape(8.dp))
+                    .padding(16.dp)
+                    .testTag("InputContainer")) {
+              Column {
+                // Row for input fields for time, category, and icon
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.spacedBy(8.dp),
+                    verticalAlignment = Alignment.CenterVertically) {
+                      // Time input field
+                      OutlinedTextField(
+                          value = stepTime,
+                          onValueChange = { stepTime = it },
+                          label = {
+                            Text(stringResource(R.string.time_label), style = Typography.bodySmall)
+                          },
+                          keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+                          colors =
+                              TextFieldDefaults.outlinedTextFieldColors(
```

Comment:
Here you use the default colors, but don't focusedLabelColor and unfocusedLabelColor. This leads to the label disappearing when text is inputted.

In this case, it's the instruction text label:
![image](https://github.com/user-attachments/assets/9bf82486-d14a-40ff-841e-099dbb03f961)

Reply:
Same as before: this will be resolved in a separate PR

Reply:
Done in #194